scenario/stages/5_network_transporter.yaml

```diff
@@ -0,0 +1,15 @@
+# Stage to migrate network resources such as networks, subnets,
+ports, router, FIPs, sec groups. 
```

Comment:
+ Neutron quotas, LoadBalancer resources (pools, monitors, members, vips)
- ports (not all; f.e. for VMs we create them in `prepare_networks` task)